network/p2p/connection/connector_factory.go

```diff
@@ -0,0 +1,36 @@
+package connection
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/host"
+	discoveryBackoff "github.com/libp2p/go-libp2p/p2p/discovery/backoff"
+)
+
+// DefaultLibp2pBackoffConnectorFactory is a factory function to create a new BackoffConnector. It uses the default
+// values for the backoff connector.
+// (https://github.com/libp2p/go-libp2p-pubsub/blob/master/discovery.go#L34)
+func DefaultLibp2pBackoffConnectorFactory(host host.Host) func() (*discoveryBackoff.BackoffConnector, error) {
+	return func() (*discoveryBackoff.BackoffConnector, error) {
+		rngSrc := rand.NewSource(rand.Int63())
+		minBackoff, maxBackoff := time.Second*10, time.Hour
+		cacheSize := 100
+		dialTimeout := time.Minute * 2
+		backoff := discoveryBackoff.NewExponentialBackoff(
+			minBackoff,
+			maxBackoff,
+			discoveryBackoff.FullJitter,
+			time.Second,
+			5.0,
+			0,
```

Comment:
Add default const values for these inline params

Reply:
https://github.com/onflow/flow-go/pull/4254/commits/fa533903d688c3ec118c5080826046b3e4fc73bb